contributing.md

```diff
@@ -0,0 +1,30 @@
+# Contribution Guidelines
+This contribution document states the general rules for contributing to this project.  
+[Bust-A-Move Group 66 GitHub page](https://github.com/iiKoe/bustamove)
+
+### General Rules
+**(1)** Submissions are allowed only after checking that Gradle builds correctly, and no tool errors exist.
+
+**(2)** When submitting a commit that helps fix a stated issue, please make sure to mention the issue number in that commit in a way that Github will detect and delete the issue. If preferred, one is free to delete the issue manually.
+
+**(3)** When in doubt, feel free request changes to any pull. Pulls with requested changes MUST NOT be merged.
+
+### Pull Requests
+Pull requests are devided in the following 5 categories:
+
+##### (1) Normal pull requests:
+Normal pull request must be reviewed by at least 2 members of the group. This includes the addition of new classes, features, enhancements, and bug fixes that interfere with many classes; namely two other members classes. The two reviewers must check the code quality, and approve **ONLY** if everything is okay, and the submitter have responded to their comments, if any. -- If a reviewer finds necessary, he can double check that the game works completely fine on his computer i.e. all good and Gradle builds. When so, he must mention this in his feedback.
+
+##### (2) Simple pull requests:
+Simple pulls are small in size and include only trivial changes. This includes fixing issues and bugs related to the requester's own class or only one another member class. For a pull to be considered simple, the requester must mention this in the pull description, and assign one reviewer. If the request concerns only one team member code, he should be tagged. If it concerns the requester's previous code, a free team member should be tagged to approve it.
```

Comment:
Should this be assigned to a random person? We could leave it open for the first person to check on it.